app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
Your test code is clear and checks the split_integer function well. However, consider adding descriptive error messages in your assertions to help identify issues when a test fails. Great work overall!